src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDatabaseMetaData.java

```diff
@@ -1518,7 +1518,8 @@ public java.sql.ResultSet getSchemas(String catalog, String schemaPattern) throw
         if (loggerExternal.isLoggable(Level.FINER) && Util.isActivityTraceOn()) {
             loggerExternal.finer(toString() + ACTIVITY_ID + ActivityCorrelator.getCurrent().toString());
         }
-        return getSchemasInternal(catalog, schemaPattern);
+        String schema = escapeIDName(schemaPattern);
+        return getSchemasInternal(catalog, schema);
```

Comment:
Alternatively, we can just pass in `schemaPattern` and escape inside of `getSchemasInternal`.

Reply:
Added